File: typescript/euler/bin/p0015.ts
// project euler: problem 15

import { factorial } from "../lib/math.ts";

export const compute = (m: number, n: number): string => {
  return String(
    factorial(BigInt(m + n)) / factorial(BigInt(m)) / factorial(BigInt(n)),
  );
};

export const solve = (): string => compute(20, 20);

File: julia/Euler/src/bin/p0015.jl

# project euler: problem 15

module Prob0015

function solve_0015(row::Int = 20, col::Int = 20)
    # factorial(big(row + col)) ÷ factorial(row) ÷ factorial(col)
    binomial(row + col, row)
end

end #module

using .Prob0015: solve_0015
export solve_0015
